### PR TITLE
Test cleaups

### PR DESCRIPTION
**Pull request by phlax**
removing unused code and fixtures from pytest_pootle and tests

### CODE REVIEW

File: pytest_pootle/fixtures/models/store.py [hunk 1]
```diff
@@ -205,25 +205,19 @@ def _require_store(tp, po_dir, name):
...
                                       units=None,
                                       suggestion="SUGGESTION"):
 
     from pootle.core.models import Revision
 
-    from tests.models.store import _update_from_upload_file
-
     # Update store as user
-    if filename:
-        _update_from_upload_file(store, filename, user=user)
```
Comment (ta2-1) [inline pytest_pootle/fixtures/models/store.py]: don't we need test submissions with `UPLOAD` type?

Comment (phlax) [inline pytest_pootle/fixtures/models/store.py]: @ta2-1 see tests in ``tests/models/store.py`` it is tested

Comment (unho) [inline pytest_pootle/fixtures/models/store.py]: Perhaps it is cleaner to have a separate test for that.

Comment (unho) [inline pytest_pootle/fixtures/models/store.py]: Now I see that there is a test for that.

**Comment by unho** [comment]
lgtm

**Comment by ta2-1** [comment]
gtm

**Pull request merged**